frontend/cypress/integration/containers.spec.js

```diff
@@ -2,7 +2,7 @@
 
 // Fixtures:
 //  - credentials.json
-//  - Container_creation_v3_2_0_F_A_2.xlsx
+//  - Container_creation_v3_4_0_F_A_2.xlsx
```

Comment:
For the naming, the "2" was intended as an intra-version counter to differentiate between test template versino within a version of the template... It should reset to 1 if you change the template version (vx_y_z part).